test-configs.yaml

```diff
@@ -1415,6 +1445,9 @@ test_configs:
       - boot
       - simple
 
+  - device_type: meson-g12b-a311d-khadas-vim3
+    test_plans: [boot, simple]
```

Comment:
any reason why its only boot and simple and not baseline too?

NIT: most others that have two or more test_plans are on multiple rows.

Reply:
no reason. I will fix it

Reply:
Waiting for this to be resolved before merging.